marketplace_confg/chart/dicom-adapter/templates/manifest.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: "{{ .Release.Name }}-adapter"
```

Comment:
Just wondering what would this name property resolve to?

Reply:
The .Release.Name is an instance name decided by the users when they are deploying adapter on the Marketplace.